docs/metrics/diarization-error.md

```diff
@@ -0,0 +1,103 @@
+# Diarization Error Rate
+Diarization Error Rate (DER) is an important metric used to evaluate speaker diarization systems. It quantifies the overall performance of a speaker diarization system by measuring the ratio of the duration of errors to the total reference speech duration. DER is calculated as a score between 0 and 1, with 0 being a perfect match and 1 being a complete mismatch between candidate and reference diarization.
+
+## Implementation Details
+### Background
+To simplest way to quantify the error in a candidate diarization is to measure the duration of false alarms, missed detections, and speaker confusions. These three elementary errors form the building blocks of diarization error rate.
+
+??? example "False Alarm"
+    False alarm is duration of non-speech classified as speech — analagous to false positive diarizations. Using the following hypothetical reference and candidate diarization segments, let's calculate our false alarm duration.
+
+    ```python
+    from pyannote.core import Segment, Annotation
+
+    reference = Annotation()
+    reference[Segment(0, 10)] = 'A'
+    reference[Segment(13, 21)] = 'B'
+    reference[Segment(23, 32)] = 'A'
+    reference[Segment(32, 40)] = 'C'
+
+    hypothesis = Annotation()
+    hypothesis[Segment(2, 14)] = 'A'
+    hypothesis[Segment(14, 15)] = 'B'
+    hypothesis[Segment(15, 20)] = 'A'
+    hypothesis[Segment(23, 36)] = 'D'
+    hypothesis[Segment(36, 40)] = 'C'
+    ```
+
+    ![Visualization](../assets/images/diarization1.png)
+
+    Upon inspection, at time segments `[10, 13]` and `[23, 24]`, our candidate diarization contains speech that doesn't exist in the reference diarization. Thus, our false alarm is equal to $3 + 1 = 4$ seconds.
```

Comment:
There seem to be GT diarization at `[23, 24]`.

Reply:
My apologies - I've edited the code to more accurately reflect the image example (I changed the image earlier, but forgot to change the code as well)